reactor-netty-core/src/main/java/reactor/netty/udp/UdpServer.java

```diff
@@ -200,6 +200,10 @@ public final UdpServer metrics(boolean enable) {
 		return super.metrics(enable);
 	}
 
+	public final UdpServer metrics(boolean enable, String prefix) {
+		return metrics(enable, () -> new UdpServerConfig.MicrometerUdpServerMetricsRecorder(prefix));
+	}
+
 	@Override
```

Comment:
Is this needed? One can use 
`reactor.netty.transport.Transport#metrics(boolean, java.util.function.Supplier<? extends reactor.netty.channel.ChannelMetricsRecorder>)` with `MicrometerChannelMetricsRecorder`